docs/how-to-configure-new-repository.md

```diff
@@ -0,0 +1,48 @@
+# How we configure repositories
+
+## Naming
+
+Most repositories have a name pattern like this:
+`open-telemetry/opentelemetry-foo`. The reason to include `opentelemetry` in the
+name of the repository is to simplify repositories distinction in forks. So when
+One will fork the repository - fork's name will still indicate that this
+repository is from OpenTelemetry organization.
+
+## Permissions
+
+Documents [Community Membership](../community-membership.md) and
+[CONTRIBUTING](../CONTRIBUTING.md) define how permissions are typically set up
+for the repository.
+
+1. Every repository has two teams associated with it. Typically they will be
+   named `foo-approvers` and `foo-maintainers`. `foo-maintainers` is a child of
+   `foo-approvers` as it always contains subset of people and defines larger
+   scope of privileges.
+2. Both teams has `Write` permissions for the repository.
+3. Root-level `CODEOWNERS` file on the repository should include superset of
```

Comment:
I prefer this under .github/ directory. What do you think?

Reply:
hm, why hide it? It is a main source of information on who to ask about this repo. Is your concern a number of files in the root folder?

Reply:
Cleaner root directory. but this is minor, don't care where we put it, just want to enforce consistency.

Reply:
Let's keep it in the root for now then.